.github/workflows/wheels-macos.yml

```
@@ -12,44 +12,29 @@ on:
 
 env:
   REPO_DIR: Pillow
-  BUILD_DEPENDS: ""
   TEST_DEPENDS: "pytest pytest-timeout"
-  WHEEL_SDIR: wheelhouse
 
 jobs:
   build:
-    name: ${{ matrix.os-name }} ${{ matrix.python }} ${{ matrix.platform }}
-    runs-on: ${{ matrix.os }}
+    name: ${{ matrix.python }} ${{ matrix.platform }}
+    runs-on: "macos-latest"
     strategy:
       fail-fast: false
       matrix:
-        os: [ "macos-latest", "ubuntu-20.04" ]
-        python: [ "pypy3.7-7.3.8", "pypy3.8-7.3.8", "3.7", "3.8", "3.9", "3.10" ]
-        platform: [ "arm64", "x86_64", "i686" ]
-        exclude:
-          - os: "macos-latest"
-            platform: "i686"
-          - os: "macos-latest"
-            platform: "arm64"
+        python: [ "3.8", "3.9", "3.10" ]
+        platform: [ "arm64", "x86_64" ]
+        include:
+          - platform: "x86_64"
             python: "3.7"
```

Putting the Python version before the platform seems clearer to me, and is consistent with the matrix order as well as includes for Linux builds.

```suggestion
          - python: "3.7"
            platform: "x86_64"
```

---

Ok, I've pushed a commit swapping them.